packages/cli/tests/commands/deploy.spec.ts

```diff
@@ -0,0 +1,48 @@
+import { access, mkdir, writeFile, stat } from 'node:fs/promises';
+import { tmpdir } from 'node:os';
+import { dirname, join } from 'node:path';
+import { beforeAll, describe, expect, test } from 'vitest';
+import AdmZip from 'adm-zip';
+
+import { generateBundleZip } from '../../src/commands/deploy';
+
+const tmpDir = tmpdir();
```

Comment:
Should I do cleanup when using `tmpdir`? 🤔

Reply:
Nah, locally it should get garbage collected over time, and when running on CI the fs gets nuked after the run.

Reply:
@jorgemoya I have a little utility that makes tempdir's, if you want to use it for this:
https://github.com/bigcommerce/catalyst/blob/778bd214004e31383c3db5c6bfc7b52d425b8f7c/packages/cli/src/commands/build.ts#L28-L46

We can move it to a `lib` folder or something. It can be refactored to take a prefix so you can prefix test tmpdir's differently than my build one if you'd like